user-access-handlers/src/main/java/no/unit/nva/handlers/data/DefaultRoleSource.java

```diff
@@ -0,0 +1,90 @@
+package no.unit.nva.handlers.data;
+
+import static no.unit.nva.database.IdentityService.Constants.ROLE_ACQUIRED_BY_ALL_PEOPLE_WITH_ACTIVE_EMPLOYMENT;
+import static nva.commons.apigateway.AccessRight.ADMINISTRATE_APPLICATION;
+import static nva.commons.apigateway.AccessRight.APPROVE_DOI_REQUEST;
+import static nva.commons.apigateway.AccessRight.APPROVE_PUBLISH_REQUEST;
+import static nva.commons.apigateway.AccessRight.EDIT_ALL_NON_DEGREE_RESOURCES;
+import static nva.commons.apigateway.AccessRight.EDIT_OWN_INSTITUTION_PUBLICATION_WORKFLOW;
+import static nva.commons.apigateway.AccessRight.EDIT_OWN_INSTITUTION_RESOURCES;
+import static nva.commons.apigateway.AccessRight.EDIT_OWN_INSTITUTION_USERS;
+import static nva.commons.apigateway.AccessRight.MANAGE_OWN_PROJECTS;
+import static nva.commons.apigateway.AccessRight.PROCESS_IMPORT_CANDIDATE;
+import static nva.commons.apigateway.AccessRight.PUBLISH_DEGREE;
+import static nva.commons.apigateway.AccessRight.PUBLISH_DEGREE_EMBARGO_READ;
+import static nva.commons.apigateway.AccessRight.READ_DOI_REQUEST;
+import static nva.commons.apigateway.AccessRight.REJECT_DOI_REQUEST;
+import java.util.List;
+import no.unit.nva.handlers.RoleSource;
+import no.unit.nva.useraccessservice.model.RoleDto;
+
+public class DefaultRoleSource implements RoleSource {
+
+    protected static final String CURATOR_ROLE_NAME = "Curator";
+    protected static final String INSTITUTION_ADMIN_ROLE_NAME = "Institution-admin";
+    protected static final String CURATOR_IMPORT_CANDIDATE_ROLE_NAME = "Curator-Import-candidate";
+    protected static final String CURATOR_THESIS_ROLE_NAME = "Curator-thesis";
+    protected static final String CURATOR_THESIS_EMBARGO_ROLE_NAME = "Curator-thesis-embargo";
+    protected static final String APP_ADMIN_ROLE_NAME = "App-admin";
+    protected static final String EDITOR_ROLE_NAME = "Editor";
+
+    private static final RoleDto CREATOR_ROLE = RoleDto.newBuilder()
```

Comment:
Is creator the best name for a person with active employment?

Reply:
Not really, but that is what it ended up as back in the days. Someone uses the term "Registrator" too. Anyway, changing it now might not be a good idea, but an overhaul of the roles, access rights and users sometime would definitely be a good idea. I am by no means assured that we have all the access rights that we should have, and that we actually do the correct checks in our handlers. PO ofter states "Curators should be allowed to ...", and often we just pick one of the existing access right given to curators only to check. Not optimal at all.